src/styles/main.scss

```diff
@@ -1,3 +1,83 @@
+@import "./variables.scss";
+
 body {
   margin: 0;
 }
+
+* {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  gap: 1px;
+  margin-top: 100px;
+  margin-left: $general-margin;
+  margin-right: $general-margin;
+  text-align: center;
+  
+  &__elem {
+    display: block;
+    width: 98px;
+    height: 98px;
+    background-color: $color-main;
+    border: 1px solid $color-border;
+    transition: 500ms;
+  }
+
+  &__elem:hover {
+    transform: translateY(-20px);
+    transition-duration: 500ms;
+    background-color: $color-hover;
+    cursor: pointer;
+  }
+  
+  @for $day from 1 through 31 {
+    &__elem:nth-child(#{$day})::before {
+      content: "#{$day}";
+      position: relative;
+      top: 35px;
+    }
+  }
+  
+  $weekdays:
+    "mon" $one,
+    "tue" $two,
+    "wed" $three,
+    "thu" $four,
+    "fri" $five,
+    "sat" $six,
+    "sun" $seven;
+
+  @each $wday, $num in $weekdays {
+    &--start-day-#{$wday} {
+      .calendar__elem:nth-child(1) {
```

Comment:
Try this:
https://developer.mozilla.org/en-US/docs/Web/CSS/:first-child

Reply:
sure)